core/src/main/scala/org/apache/spark/ui/exec/ExecutorsPage.scala

```diff
@@ -33,11 +33,13 @@ private[ui] case class ExecutorSummaryInfo(
     rddBlocks: Int,
     memoryUsed: Long,
     diskUsed: Long,
+    totalCores: Int,
```

Comment:
So the comment for this case class says it isn't used anymore - do we really need to update it?